src/TablerPreset.php

```diff
@@ -90,4 +101,30 @@ protected static function compileControllerStub()
             file_get_contents(__DIR__.'/tabler-stubs/controllers/HomeController.stub')
         );
     }
+
+    /**
+     * Gets resource path depending on version of Laravel.
+     *
+     * @param string $path
+     *
+     * @return string
+     */
+    protected static function getResourcePath($path = '')
+    {
+        if (self::isLaravel57orUp()) {
+            return resource_path($path);
+        }
+
+        return resource_path('assets/'.$path);
+    }
+
+    /**
+     * Is running in Laravel 5.7 or up?
+     *
+     * @return bool
+     */
+    protected static function isLaravel57orUp()
```

Comment:
This function can be named `isLatestLaravelVersion()` for example, try to make it a more general name.

Reply:
I'm not sure if latest is the right way to put this. The folder structure change was implemented in 5.7. Calling this method isLatestLaravelVersion() would imply that we are always checking for the latest version (which we could need to actively change the code for on each release). We only want to know if the version being used is 5.7 or above.

Reply:
Perhaps this could be changed to something like hasAssetsFolder() instead?

Reply:
Great, it seems more appropriate to me that way.